src/server/api/controllers/getAllUsers.controller.js

```diff
@@ -0,0 +1,16 @@
+const knex = require('../../config/db');
+
+const getAllUsers = async () => {
+  try {
+    return await knex('game_scores')
+      .select('game_scores.fk_user_id', 'game_scores.score')
+      .orderBy('game_scores.score', 'desc')
+      .limit(2);
```

Comment:
You need to show all the results, by putting limit it will hide all user except first two.

Reply:
Limit has been deleted.